rllib/algorithms/algorithm_config.py

```diff
@@ -792,6 +815,11 @@ def validate(self) -> None:
                 # compatibility for now. User only needs to set num_rollout_workers.
                 self.input_config["parallelism"] = self.num_rollout_workers or 1
 
+        # resolve rl_module class
+        if self._enable_rl_module_api and self.rl_module_class is None:
+            rl_module_class = self.get_rl_module_class(framework_str=self.framework_str)
```

Comment:
ack on renaming. User may create the new RLModule right on the spot. So it may be just easier to pass in the type since it's already available instead of figuring out the path.